run/README.namelist

```diff
@@ -77,7 +77,8 @@ Description of namelist variables
  adjust_output_times                 = .false., ! adjust output times to the nearest hour
  override_restart_timers             = .false., ! whether to change the alarms from what is previously set
  write_hist_at_0h_rst                = .false., ! whether to output history file at the start of restart run
- output_ready_flag		     = .true.,  ! asks the model to write-out an empty file with the name 'wrfoutReady_d<domain>_<date>.  
+ output_ready_flag		     = .true.,  ! asks the model to write-out an empty file with the name 
+                                                  'wrfoutReady_d<domain>_<date>.  
```

Comment:
What is the purpose of moving these to the next line? I know it helps with wrapping, but that will still be dependent on the user's window. If I decrease the size of this window, even this begins to wrap.

Reply:
@kkeene44 True, but I just like to make the width of the text similar inside the file.